transport/control.go

```diff
@@ -46,8 +46,8 @@ const (
 	// The default value of flow control window size in HTTP2 spec.
 	defaultWindowSize = 65535
 	// The initial window size for flow control.
-	initialWindowSize             = defaultWindowSize      // for an RPC
-	initialConnWindowSize         = defaultWindowSize * 16 // for a connection
+	initialWindowSize             = defaultWindowSize * 2048      // for an RPC
+	initialConnWindowSize         = defaultWindowSize * 16 * 2048 // for a connection
```

Comment:
> a change which left the defaults unchanged but made the stream and window sizes configurable

I backed the multiplier back to 64x (4MB). I tried 32x but that was still loosing 8% compared to 64x on my 2.7msec latency link.

Options are great, but the defaults should be sane too. I would think. Because most users won't ever dig deep enough to discover the option.

> you've likely hurt latency for small requests sharing the same connection

This comment I don't follow the reasoning, but I would like to. How would latency for small requests sharing the same connection be hurt by a bigger window; I don't see how their frames would be delayed since they are likely to be sent and reach the receiver sooner.